src/cbc_syslog/__init__.py

```diff
@@ -14,8 +14,8 @@
 __title__ = 'cbc_syslog'
 __author__ = 'Carbon Black Developer Network'
 __license__ = 'MIT'
-__copyright__ = 'Copyright 2018-2023 VMware Carbon Black'
-__version__ = '2.0.3'
+__copyright__ = 'Copyright 2018-2024 VMware Carbon Black'
```

Comment:
Does this need modified to the new string

Reply:
Fixing that.